theme/style.css

```diff
@@ -43,28 +44,29 @@ img {
   background-repeat: no-repeat;
   background-size: cover;
   background-position: center;
-  height: 500px;
   border-radius: 20px;
   padding: 20px;
+  min-width: 1750px;
```

Comment:
this is going to clobber the grid. Make sure that you're using flex rather than setting a fixed width that's wider than most screens!